pkg/archive/archive_test.go

```diff
@@ -724,6 +725,77 @@ func TestTarUntar(t *testing.T) {
 	}
 }
 
+func TestTarWithOptionsChownOptsAlwaysOverridesIdPair(t *testing.T) {
```

Comment:
About this test method name I'm still unsure, there's already an existing `TestTarWithOptions` method but merging my tests inside it would looks like a big mess, so illustrating real use cases with more splitted methods was looking good to me. WDYT?